eng/build.ps1

```diff
@@ -43,7 +43,7 @@ function Get-Help() {
 
   Write-Host "Libraries settings:"
   Write-Host "  -vs                     Open the solution with VS for Test Explorer support. Path or solution name (ie -vs Microsoft.CSharp)"
-  Write-Host "  -framework              Build framework: netcoreapp or netfx (short: -f)"
+  Write-Host "  -framework              Build framework: netcoreapp or net472 (short: -f)"
```

Comment:
Shouldn't this be netcoreapp5.0 instead?

Reply:
Ha, good catch.

Reply:
Actually there are more possible build frameworks but documenting the latest ones should be sufficient.

Reply:
Yeah. I guess this are just examples, so it makes sense to document the latest.